### PR TITLE
added docs for issue flow

### PR DESCRIPTION
**Pull request by joester89**

### CODE REVIEW

File: docs/issues/Issues.md [hunk 1]
```diff
@@ -0,0 +1,37 @@
+#Issues
+
+##Labels
+All labels available: https://github.com/HeRAMS-WHO/herams-backend/labels
+
+##Creating an issue
+###Description
+When creating an issue be descriptive as possible. A good approach is:
```
Comment (SamMousa) [inline docs/issues/Issues.md:8]:
```suggestion
When creating an issue be as descriptive as possible. A good approach is:
```

File: docs/issues/Issues.md [hunk 2]
```diff
@@ -0,0 +1,37 @@
...
+
+"I am logged in as `...` with `...`, and am on page `...`. I want to `...`"
```
Comment (SamMousa) [inline docs/issues/Issues.md:10]: We should use issue templates instead of relying on the user.

File: docs/issues/Issues.md [hunk 3]
```diff
@@ -0,0 +1,37 @@
...
+
+From this description it should be clear for developers to reproduce the taken steps to get to the problem.
+The desired next steps and features should be just as clear.
+
+###Assigning labels
+Labels must be assigned to make it very clear what the status of the issue is. Use the following issue categories:
+* **env** Indicate on which platform the error occurs, not needed when it is a new feature  
+* **urgency** When the urgency is other than normal, assign an urgency label like _extreme_, _high_ or _low_
```
Comment (SamMousa) [inline docs/issues/Issues.md:18]: Default should be low. We could even automate this.

File: docs/issues/Issues.md [hunk 4]
```diff
@@ -0,0 +1,37 @@
...
+* **subject** Help developers to filter issues to pick up quickly by adding a subject
+* **type** Help developers prioritize issues withing an urgency, i.e. bug often is picked up before an enhancement
+
+##Issue flow
+###Assignee
+The developers will assign the issues to themselves when they start working on an issue.
+
+###Flow
+* When there is **an assignee**, it can be assumed the issue is in progress
```
Comment (SamMousa) [inline docs/issues/Issues.md:27]: This is currently not true, an issue can also be assigned back to the reporter or other stakeholders. In which case development on it is not in progress.

File: docs/issues/Issues.md [hunk 5]
```diff
@@ -0,0 +1,37 @@
...
+* When the implementation is complete and released on staging, a developer will add **flow:test on staging**
```
Comment (SamMousa) [inline docs/issues/Issues.md:28]: We should mark this as an automation. By using proper commit messages we can find which issues should get the status after the automated release.

File: docs/issues/Issues.md [hunk 6]
```diff
@@ -0,0 +1,37 @@
...
+* It must then be checked by the reporter on staging:
```
Comment (SamMousa) [inline docs/issues/Issues.md:29]: Due to the load on @luna404 for small issues we should self test and self report. Maybe, when @luna404 is the reporter she should mark if she wants to be involved in testing the solution or not. (For minor issues it is reasonable to make it a fire & forget)

File: docs/issues/Issues.md (hunk 3 above)
Comment (joester89) [reply docs/issues/Issues.md:18]: I would say you want to use low to indicate an even lower as normal priority.

File: docs/issues/Issues.md (hunk 2 above)
Comment (joester89) [reply docs/issues/Issues.md:10]: Didn't know Github had such things but would defenitely be the next step

**Pull request merged**